app/src/main/java/org/mozilla/fenix/settings/sitepermissions/SitePermissionsExceptionsFragment.kt

```diff
@@ -44,6 +45,7 @@ class SitePermissionsExceptionsFragment :
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        (activity as? HomeActivity)?.getSupportActionBarAndInflateIfNecessary()
```

Comment:
Same comments as others – we really don't want anyone calling for `supportActionBar`.